src/expr/src/scalar/func.rs

```diff
@@ -1917,14 +1917,12 @@ where
         Err(EvalError::DateBinOutOfRange(
             "timestamps cannot be binned into intervals containing months or years".to_string(),
         ))
-    } else if stride.duration <= 0 {
+    } else if stride.micros <= 0 {
```

Comment:
Note to self: These need to include days. Probably should combine days and times here, check PG.